### ISSUE TITLE
Top serif on lowercase i could be longer

### ISSUE THREAD

**Issue by yannakopoulos** [source-foundry/Hack#463]
When compared to the top serifs on similarly-shaped `l` and `j` and even the left arm of `t`, the top serif on `i` seems incongruously short. Might it be extended to match the others?

**Comment by chrissimpkins** [comment]
Mind posting some images that demonstrate what you are describing?  These issues are most easily addressed visually.

**Comment by yannakopoulos** [comment]
Sure, here's an example. Check out the relative endings of the serifs on `i` compared to the letters directly above/below them. 

![image](https://user-images.githubusercontent.com/9258329/48651076-31ea4300-e9c7-11e8-8f6a-9d105098bc20.png)

Here's the same image with `i` overlaying some letters as best as I could arrange it on a difference layer, showcasing the longer serif (here the letters are black where they overlap and white where they don't.) 

![image](https://user-images.githubusercontent.com/9258329/48651277-2b100000-e9c8-11e8-9a10-e3a902816a24.png)

**Comment by chrissimpkins** [comment]
Yes will definitely take a look at this.  Thanks for the images.  I drew the `i`and don't remember if this was intentional or not.  There may have been a rationale when positioned next to other glyphs.  If there was I don't recall :)

Do you happen to draw? If so you are welcome to submit a PR and we can build it.

**Comment by yannakopoulos** [comment]
Sorry, I have no experience with drawing. Thanks for checking it out!
